### PR TITLE
updated apply info

### PR DESCRIPTION
**Pull request by josephz-me**

### CODE REVIEW

**Comment by mikinty** [comment]
Can you post a screenshot of the changes please?

Also I'm not going to be super strict on this, but would be nice to separate out the formatting changes and the new content changes to separate commits

File: pages/apply.html [hunk 1]
```diff
@@ -1,210 +1,323 @@
...
+                <a
+                  href="https://calendar.google.com/event?action=TEMPLATE&tmeid=YWkxaDFkdjVwYW5xNWR2ZWQzMGFwYTg3YzQganRpbWN6eWtAYW5kcmV3LmNtdS5lZHU&tmsrc=jtimczyk%40andrew.cmu.edu"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  >Google Calendar</a
+                >
+              </p>
+            </div>
+          </div> -->
+        <!-- </div> -->
```
Comment (mikinty) [inline pages/apply.html:135]: Can you just delete this? Not need to comment with version control

File: pages/apply.html [hunk 2]
```diff
@@ -1,210 +1,323 @@
...
-              <br><a href="https://cmu.zoom.us/j/96709488615?pwd=Q3FlUURwUGFPRnVOZ0tWMEduMTcyQT09" target="_blank" rel="noopener noreferrer">Zoom</a> / Pass: 662304
-            </p>
+                <br />
+                <!-- <a
+                  href="https://cmu.zoom.us/j/96709488615?pwd=Q3FlUURwUGFPRnVOZ0tWMEduMTcyQT09"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  >Zoom</a
+                > -->
+                <!-- / Pass: 662304 -->
```
Comment (mikinty) [inline pages/apply.html:231]: Again, no need to comment this out, just delete it.

Also reminder that comments still appear on page sources, so people can find this if they inspect element

File: pages/apply.html [hunk 3]
```diff
@@ -1,210 +1,323 @@
...
+            </div>
+            <div class="col-3-xl ml-lg-3 my-1">
+              <p>
+                Video Submission Deadline
+                <br />
+                Oct.4 <span id="atSign">@</span> 11PM EST <br />
+                Email submissions to entertainment@lunargala.org
+              </p>
+            </div>
           </div>
```
Comment (mikinty) [inline pages/apply.html:272]: Delete

**Review by mikinty — CHANGES_REQUESTED** [review]

**Comment by josephz-me** [comment]
> Can you post a screenshot of the changes please?
> 
> Also I'm not going to be super strict on this, but would be nice to separate out the formatting changes and the new content changes to separate commits

**Comment by josephz-me** [comment]
Sounds good, forgot to turn off my formatter when I made these changes — only copy changes were made.

**Pull request closed without merging**